pkg/dir.props

```diff
@@ -28,7 +28,7 @@
   <PropertyGroup>
     <PackageOutputPath>$(PackagesOutDir)</PackageOutputPath>
     <SymbolPackageOutputPath>$(SymbolPackagesOutDir)</SymbolPackageOutputPath>
-    <RuntimeIdGraphDefinitionFile>$(ProjectDir)packages\Microsoft.NETCore.Platforms\1.0.2\runtime.json</RuntimeIdGraphDefinitionFile>
+    <RuntimeIdGraphDefinitionFile>$(ProjectDir)packages\Microsoft.NETCore.Platforms\DEPENDENCY 'Microsoft.NETCore.Platforms' NOT FOUND\runtime.json</RuntimeIdGraphDefinitionFile>
```

Comment:
I should have time to work on that starting around noon PST. (Same fix for both branches.)